tests/files/validate-ci.yml

```diff
@@ -17,10 +17,6 @@
 - name: Validate zuul status
   hosts: zuul
   tasks:
-    - name: zuul webapp is running
-      uri:
-        url: http://localhost:8001/status.json
```

Comment:
yea, the new url is /{tenant_name}/status.json - but we don't have a default tenant we can assert the test with. Was trying to think of some way of doing it conditionally, or assume a tenant, and then just removed it.